lib/npg_qc/autoqc/checks/pulldown_metrics.pm

```diff
@@ -116,16 +140,22 @@ override 'execute' => sub {
         return 1;
     }
 
-    $self->result->set_info( 'Aligner', qq[Picard $PICARD_JAR_NAME] );
-    $self->result->set_info( 'Aligner_version', $self->current_version($self->picard_jar_path) );
+    $self->result->set_info( 'Aligner', 'Picard '.$self->picard_module );
+    $self->result->set_info( 'Aligner_version', $self->current_version($self->gatk_cmd) );
     $self->result->bait_path($self->bait_path);
-
-    my $command = $self->picard_command;
-    ## no critic (ProhibitTwoArgOpen InputOutput::RequireBriefOpen)
-    open my $fh, "$command |" or croak qq[Cannot fork "$command". $ERRNO];
+    local $ENV{JAVA_TOOL_OPTIONS} = '-Xmx'.$self->max_java_heap_size;
```

Comment:
The [recommended way](https://gatk.broadinstitute.org/hc/en-us/articles/360035531892-GATK4-command-line-syntax) seems to be `gatk --java-options "-Xmx4G"`. Does it work for picard?

Reply:
Tried that. System does funny things to arg lists when invoking java. I either get `--java-options` is not a valid Jar or `gatk --java-options` is not a valid command. There's probably a way around it but I think I wasted quite enough time trying already.

Reply:
`perl -le 'print system q[gatk --java-options "-Xmx4G"]` works.
`perl -le 'print "SYSTEM EXIT CODE " . system q[gatk --java-options "-Xmx1G" CollectHsMetrics --BAIT_INTERVALS=dodo --TARGET_INTERVALS=dodo --INPUT foo.bam --OUTPUT foo.txt]'` works as well in a sense that it displays an error message for this particular picard tool